### PR TITLE
add_dropdown_menu

### PR DESCRIPTION
**Pull request by Mike47ip**
add_dropdown_menu to he mobile version of the portfolio

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -48,23 +48,27 @@ footer a {
...
   width: 100%;
   border: 1px solid #28352f;
-}
+} */
 
-.vector span:nth-child(2) {
+/* .vector span:nth-child(2) {
   margin: 2px 0;
-}
+} */
```
Comment (ishimwezachee) [inline style.css]: - [x]  You should not commit inline comments. They will make your project look messy.  remove all commented codes in your project to have clear and clean source codes.

**Review by ishimwezachee — CHANGES_REQUESTED** [review]
Hi @Mike47ip  and @paulshaun-kdtk, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
image

## Highlights
✅ Well organized project 
✅ Github flow is well implemented 
✅ project is working as expected 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Mike47ip** [comment]
Dear reviewer kindly ignore the webhint error. There seem to be a bugging issue with the linters. Technical support advice that we ignore.

ERR REQUIRE ESM

**Review by ShadyShawkat — APPROVED** [review]
## Hi @Mike47ip, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- All linters are passing ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Mike47ip** [comment]
@ShadyShawkat bro I cannot go on to submit the project as completed after you approved it. There seem to be something wrong. Please advice what i do now

**Pull request merged**